cprover_bindings/src/goto_program/expr.rs

```diff
@@ -1013,6 +1036,16 @@ impl Expr {
         expr!(BinOp { op, lhs: self, rhs }, Expr::binop_return_type(op, &self, &rhs))
     }
 
+    /// Like `binop`, but receives an additional parameter `ret_typ` with the expected
+    /// return type for the place, which is used as the return type.
+    pub fn vector_cmp(self, op: BinaryOperator, rhs: Expr, ret_typ: Type) -> Expr {
+        assert!(
+            Expr::typecheck_vector_cmp_expr(&self, &rhs, &ret_typ),
+            "vector comparison expression does not typecheck",
```

Comment:
nit: Can you print `rhs` and `ret_typ` here? It makes it a huge difference when debugging a failure.

Reply:
Done. Prints `lhs`, `rhs` and `ret_typ` on failure.